addons/mail/static/src/models/partner_im_status_icon_view.js

```diff
@@ -0,0 +1,83 @@
+/** @odoo-module **/
+
+import { one } from '@mail/model/model_field';
+import { registerModel } from '@mail/model/model_core';
+import { clear, replace } from '@mail/model/model_field_command';
+
+registerModel({
+    name: 'PartnerImStatusIconView',
+    identifyingFields: [[
+        'channelInvitationFormSelectablePartnerViewOwner',
+        'channelMemberViewOwner',
+        'composerSuggestionViewOwner',
+        'messageViewOwner',
+        'notificationRequestViewOwner',
+        'threadNeedactionPreviewViewOwner',
+        'threadPreviewViewOwner',
+    ]],
+    recordMethods: {
+        /**
+         * @private
+         * @returns {FieldCommand}
+         */
+        _computePartner() {
+            if (this.channelInvitationFormSelectablePartnerViewOwner) {
+                return replace(this.channelInvitationFormSelectablePartnerViewOwner.partner);
+            }
+            if (this.channelMemberViewOwner) {
+                return replace(this.channelMemberViewOwner.channelMember.partner);
+            }
+            if (this.composerSuggestionViewOwner) {
+                return replace(this.composerSuggestionViewOwner.partner);
+            }
+            if (this.messageViewOwner) {
+                return replace(this.messageViewOwner.message.author);
+            }
+            if (this.notificationRequestViewOwner) {
+                return replace(this.messaging.partnerRoot);
```

Comment:
There is not necessary a `partnerRoot`, especially in frontend (public livechat).

Reply:
Notification request template is currently not guarding for `partnerRoot` either.

Reply:
Ok, ignore then. We can fix that later when it will be indeed needed in public livechat.
For now, only models are available in public livechat, not components. We'll fix it this part of code at the moment we want components in public livechat.